collection-scripts/gather

```diff
@@ -43,3 +43,6 @@ done
 
 # Gather Service Logs (using a suplamental Script); Scoped to Masters.
 /usr/bin/gather_service_logs master
+
+# Gather etcd information
+/usr/bin/gather_etcd
```

Comment:
Should this follow `gather_resources` (line 39)?

Reply:
I don't see a big difference so I can just move it up.